Description: remove "scary notice" from POD
 Adriano Ferreira is now the official maintainer and there's nothing unofficial
 about his Term-Size distribution any more. Sadly, he seems to have lost
 interest in Term::Size, not touching the module or this now-obsolete notice
 since 2008.
Author: Florian Schlichting <fschlich@zedat.fu-berlin.de>

--- a/Size.pm
+++ b/Size.pm
@@ -26,21 +26,6 @@
 
 =head1 DESCRIPTION
 
-  THIS IS AN UNOFFICIAL PATCH AGAINST Term-Size 0.2 DISTRIBUTION 
-  FOUND ON CPAN (http://search.cpan.org/~timpx/Term-Size-0.2/).
-  IT IS UNOFFICIAL IN THE SENSE THAT THE AUTHOR Tim Goodwin 
-  HASN'T APPROVED IT (YET, I HOPE). BECAUSE OF THIS, THIS 
-  DISTRIBUTION IS NOT INDEXED AND AVAILABLE VIA cpan OR cpanp 
-  SHELLS UNLESS YOU EXPLICITLY SAY 
-  "install FERREIRA/Term-Size-0.203.tar.gz". 
-  
-  THIS IS UNDELICATE? I THINK IT IS IN A CERTAIN SENSE. BUT IT 
-  IS A WAY TO UNFREEZE THE CURRENT DISTRIBUTION STATUS. IF TIM 
-  DISAPPROVES, I WILL REMOVE THIS DISTRIBUTION RIGHT AWAY. 
-  IF HE APPROVES, I WILL REMOVE THIS DISTRIBUTION RIGHT AWAY 
-  AND WORK OUT (AFTER BEEN GIVEN MAINTAINERSHIP STATUS) 
-  A DISTRIBUTION WITHOUT THIS NOTE AND WHICH INDEXES CORRECTLY.
-
 B<Term::Size> is a Perl module which provides a straightforward way to
 retrieve the terminal size.
 
